apps/volunteer/shifts/entrance.py

```diff
@@ -0,0 +1,30 @@
+from ..event_date import edt
+
+entrance_shifts = {
+    "Entrance Steward": {
+        "Entrance Tent": [
+            {
+                "first": edt(d, "09:00:00"),
+                "final": edt(d, "23:00:00"),
+                "min": 3,
+                "max": 6,
+            } for d in ["thur", "fri"]
+        ] +
+        [
+            {
+                "first": edt(d, "09:00:00"),
+                "final": edt(d, "21:00:00"),
+                "min": 1,
+                "max": 2,
+            } for d in ["sat", "sun"]
+        ] +
+        [
+            {
```

Comment:
I figured it would be useful to have someone on entrance tent to answer questions since there will be a lot of traffic passing entrance as folks pack up and leave. So more of info role than handling arrivals - probably better to put it in actually as info role? Or not needed and folks can just go to info desk?

Reply:
Info desk was dead Monday when I was packing things up Monday morning last time, the only people coming in were asking about teardown help. There'll be security on the gate who we can brief to get anyone who needs help to call the info desk from the phone there.